### ISSUE TITLE
Allow custom heights (non-squares)

### ISSUE THREAD

**Issue by jimbol** [BugiDev/react-native-calendar-strip#222]
Hi there,
I'm interested in making a tall `dayComponent` element. It looks like there isn't a good way to do this yet. The code says to [`assume square element sizes`](https://github.com/BugiDev/react-native-calendar-strip/blob/70e60fb7a53fc25784f666a2ff41b16febdeb3a2/src/CalendarStrip.js#L357). 

One way to make this work would be to allow a number or an object to be passed by the user.
```
// So you could do
maxDayComponentSize={80}

// or
maxDayComponentSize={{
    height: 80,
    width: 300,
}}
```

Another approach would be to change the height of the element to a min-height. This wouldn't change the API at all, which is nice.
```
<View style={[this.props.innerStyle, { height: this.state.height }]}>
```
([source](https://github.com/BugiDev/react-native-calendar-strip/blob/70e60fb7a53fc25784f666a2ff41b16febdeb3a2/src/CalendarStrip.js#L543))

**Comment by jimbol** [comment]
Pinging this and its corresponding PR, @peacechen

**Comment by AlanQuadros** [comment]
Same problem, any solutions?

**Comment by AlanQuadros** [comment]
I put minHeight: 75 in the View in Scroller.js.
I'm waiting for a prop for this.

**Comment by Sascha195** [comment]
Unfortunately I also encountered this problem. Is there already a solution for this?

**Comment by jimbol** [comment]
Theres a branch with a `dayComponentHeight` prop that I've been using (Here: https://github.com/BugiDev/react-native-calendar-strip/pull/223). Gonna try and get it merged.

**Comment by peacechen** [comment]
@jimbol Would you mind updating your PR based on the discussion in that thread?

**Comment by pedrobslisboa** [comment]
@peacechen Could you review @jimbol PR? It's a necessary feature.

What can us make to get it merged?

**Comment by pedrobslisboa** [comment]
#273 I've made a PR about it and I will keep my attention on it so maybe we can add this feature.

**Comment by pedrobslisboa** [comment]
Solved here: #273 

Thank you @peacechen for your help 🎉

**Comment by peacechen** [comment]
Published in 2.1.8
